spec/money_column_spec.rb

```diff
@@ -44,6 +49,27 @@ class MoneyWithReadOnlyCurrency < ActiveRecord::Base
     expect(record.price_usd).to eq(Money.new(1.44, 'USD'))
   end
 
+  it 'returns money when the currency is invalid' do
```

Comment:
Test name could be more descriptive, it doesn't mention that the money object's currency is the null currency when. Also shouldn't the test also assert that we can read the amount fine still?